proto/buoyant-cloud-api.proto

```diff
@@ -92,11 +92,69 @@ message Event {
   Workload owner = 3;
 }
 
+//
+// Linkerd Messages
+//
+
+// This message represents PEM encoded certificate
+// data. It might be a single certificate, a pool of
+// roots or a chain of certificates.
+message CertData {
+  bytes raw = 1;
+}
+
+// Represents the certificates that the control plane
+// has been configured with.
+message ControlPlaneCerts {
+  // This is the identity issuer certificate chain
+  CertData issuer_crt_chain = 1;
+  // The roots that the control plane has been configured with
+  CertData roots = 2;
```

Comment:
Just to put a finer point on this: are these the roots that the control plane will add to newly created proxies? Or the root (singular) that signed the issuer? Or something else?

Reply:
This is the roots that the control plane will add to newly created proxies - the roots that Linkerd has been configured with (which should be on the config map).

Reply:
Yeah. Perhaps worth calling that out in more detail in the comment?